jitbuilder/control/Jit.cpp

```diff
@@ -144,7 +144,6 @@ initializeJitBuilder(TR_RuntimeHelper *helperIDs, void **helperAddresses, int32_
       return false;
 
    initializeCodeCache(fe.codeCacheManager());
-
```

Comment:
i would revert this stylistic change that's unrelated to your PR and the only change in this file.

Reply:
Thank you for comment, I will definitely do so. There is no need to do this change, indeed.